### DISCUSSION TITLE
A word of Thanks

### DISCUSSION THREAD

**Discussion by ghost** [podman-desktop/podman-desktop#6680] [General]
I have spent a lot of hours trying out various projects in the past few months for my homelab to test some custom workloads.... OpenStack (Devstack), k3s, rancher, portainer, minikube, docker desktop, etc. and I have found what I needed with your Podman Desktop.
I can use my GPU on the podman-machine through WSL with nvidia's toolkit, AND run openshift local crc on a hyper-v vm for non gpu stuff. Then... add in support and/or extensions for all the other projects with a gui to orchestrate everything and what else could you ask for?

Thanks!

**Comment by cdrage** [comment]
Thank you so much for the thanks! We aim to make the developer experience awesome 🚀 

If you ever have any suggestions or improvements for PD, we're all open ears for it! Glad you're liking the experience!

**Comment by veillard** [comment]
Hi Mark, something you said really scratches an itch, and I would love if you would agree to provide more info (eventually privately (veillard@redhat.com), it's the GPU support. What is your use case ? An obvious one is running LLM on your desktop, but that may be something else. Would you mind sharing ? I didn't realize that Hyper-V didn't expose the GPU to the linux kernel, maybe something we could dig, it's possible crc is missing an option to activate the  GPU passthrough.

**Comment by gbraad** [comment]
> I can use my GPU on the podman-machine through WSL with nvidia's toolkit,

Would you be willing to share some notes n how to enable this? (note: Perhaps a plugin can be made to automate these steps)

> run openshift local crc on a hyper-v vm for non gpu stuff.

It might not be easy, but should we have GPU support for Hyper-V VMs of OpenShift Local? An example of a workload might help us to consider/enable this

Note: manager/team lead of CRC/OpenShift Local here.
